README.md

```diff
@@ -5,14 +5,14 @@
 
 ## Why?
 
-* Currently, the `scrapy bench` option present just spawns a spider which aggresively crawls randomly generated links at a high speed. 
+* Currently, the `scrapy bench` option present just spawns a spider which aggresively crawls randomly generated links at a high speed.
 * The speed thus obtained, which maybe useful for comparisons, does not actually reflects a real-world scenario.
 * The actual speed varies with the python version and scrapy version.
 
-### Current Features 
+### Current Features
 * Spawns a CPU-intensive spider which follows a fixed number of links of a static snapshot of the site [Books to Scrape](http://books.toscrape.com/index.html).
 * Follows a real-world scenario where various information of the books is extracted, and stored in a `.csv` file.
-* A broad crawl benchmark that uses 1000 copies of the site [Books to Scrape](http://books.toscrape.com/index.html) which are dynamically generated using `twisted`. The server file is present [here](https://github.com/scrapy/scrapy-bench/blob/broad/server.py).
+* A broad crawl benchmark that uses 1000 copies of the site [Books to Scrape](http://books.toscrape.com/index.html) which are dynamically generated using `twisted`. The server file is present [here](https://github.com/scrapy/scrapy-bench/server.py).
```

Comment:
The old link is wrong, that's a good catch, thanks! But the new one also gives 404, maybe "blob" is missing?

Reply:
The new link has `/blob/master`.

Reply:
to be clear, I mean this link ``https://github.com/scrapy/scrapy-bench/server.py`` at the end of line 15

Reply:
Yes, so the new link is `https://github.com/scrapy/scrapy-bench/blob/master/server.py`. 
The broken link is because it was initially on another branch called `broad`.